core/test/utils/api.js

```diff
@@ -63,17 +63,21 @@ function isISO8601(date) {
 function checkResponseValue(jsonResponse, expectedProperties) {
     var providedProperties = _.keys(jsonResponse),
         missing = _.difference(expectedProperties, providedProperties),
-        unexpected = _.difference(providedProperties, expectedProperties);
+        unexpected = _.difference(providedProperties, expectedProperties),
+        expectedPropertiesLength = expectedProperties.length;
 
     _.each(missing, function (prop) {
-        jsonResponse.should.have.property(prop);
+        jsonResponse.should.not.have.property(prop);
     });
 
     _.each(unexpected, function (prop) {
-        jsonResponse.should.not.have.property(prop);
+        if (prop !== 'feature_image' && prop !== 'profile_image' && prop !== 'cover_image') {
+            jsonResponse.should.not.have.property(prop);
+        } else {
+            expectedPropertiesLength += 1;
+        }
```

Comment:
It looks like we can add this to our lists on top, but these properties only occur if the image is set. That's why it's conditional. I would leave this as is.